### PR TITLE
Reduce/remove timeout on play/pause click

### LINKED ISSUE #74: Reduce/remove timeout on play/pause click
The current timeout of 1 second makes the button seem unresponsive. And if you click the button during the timeout, the click won't be registered.

We can either experiment with setting the timeout lower or implement some logic to switch between the play and pause button some other way (using setInterval and PlayerState?) .

### PR DESCRIPTION
**Pull request by monicacheung**
Closes #74

### CODE REVIEW

File: js/everything.js [hunk 1]
```diff
@@ -22,9 +22,15 @@ function onYouTubeIframeAPIReady() {/* jshint ignore:line */
                 //console.log("State changed to " + event.data);
                 var playerState = event.data;
 
-                 if (playerState === YT.PlayerState.ENDED) {
+                if (playerState === YT.PlayerState.ENDED) {
```
Comment (shakeelmohamed) [inline js/everything.js]: Can we replace this with a switch/case?

Comment (monicacheung) [inline js/everything.js]: Okay, how can I update my diff in the pull request?

Comment (shakeelmohamed) [inline js/everything.js]: You can make another commit, hop in the gitter chat if you need anything :smile:

File: js/everything.js [hunk 2]
```diff
@@ -22,9 +22,17 @@ function onYouTubeIframeAPIReady() {/* jshint ignore:line */
                 //console.log("State changed to " + event.data);
                 var playerState = event.data;
 
-                 if (playerState === YT.PlayerState.ENDED) {
-                    showPlayButton();
-                 }
+                switch (playerState) {
+                    case YT.PlayerState.ENDED:
```
Comment (shakeelmohamed) [inline js/everything.js]: We can do slightly better here using fall-through so we don't have to duplicate code for `ENDED` and `PAUSED`. See this [SO question](http://stackoverflow.com/questions/13207927/switch-statement-multiple-cases-in-javascript#13207939) for an example.

Also, we should probably have a default condition while we're at at it.
We're currently not addressing these cases.
- `YT.PlayerState.BUFFERING`
- `YT.PlayerState.CUED`

Both of which I believe should have `showPlayButton();` as their default behavior. So, maybe we can simplify this whole thing quite a bit - just have 1 case for `YT.PlayerState.PLAYING`, and a default to `showPlayButton();`

Does that make sense?

Comment (monicacheung) [inline js/everything.js]: Yup. Good catch :)

**Comment by shakeelmohamed** [comment]
Thank you @monicacheung! Welcome to the org :rocket:
